style.css

```diff
@@ -0,0 +1,252 @@
+  width: 100%;
+  text-align: center;
+  padding-top: 65px;
+  color: aliceblue;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+ }
+
+ h3 {
+  color: aliceblue;
+  text-align: center;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+ }
+
+ .flex-container {
+  margin: 10px;
+  max-width: 1400px;
+  flex-wrap: nowrap;
+  box-sizing: border-box;
+  
+}
+
+.flex-item1 {
+ padding: 10px;
+ border: 2px solid #9aedfc;
+ margin: 10px auto;
+}
+
+.flex-item2 {
+  border: 2px solid deeppink;
+  padding: 10px;
+  margin: 10px auto;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+}
+
+.form-group {
+  margin-bottom: 15px;
+  display: flex;
+  align-items: center;
+  color: aliceblue;
+  border: none;
+  
+}
+
+.form-group label {
+  width: 100px;
+  margin-right: 5px;
+}
+
+.form-group input,
+.form-group textarea {
+  width: 200px;
+}
+
+.form-submit {
+  width: 100px;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  border-radius: 4px;
+  background-color: #184b52;
+  border: none;
+  color: aliceblue;
+  text-align: center;
+  font-size: 28px;
+  padding: 20px;
+  width: auto;
+  transition: all 0.5s;
+  cursor: pointer;
+  margin: 5px;
+}
+
+.form-submit span {
+  cursor: pointer;
+  display: inline-block;
+  position: relative;
+  transition: 0.5s;
+}
+
+.form-submit span:after {
+  content: '\00bb';
+  position: absolute;
+  opacity: 0;
+  top: 0;
+  right: -20px;
+  transition: 0.5s;
+}
+
+.form-submit:hover span {
+  padding-right: 25px;
+}
+
+.form-submit:hover span:after {
+  opacity: 1;
+  right: 0;
+}
+
+h2 {
+  color: aliceblue;
+  padding: 10px;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  text-align: center;
+}
+
+
+.accordion {
+  width: 90%;
+  max-width: 1000px;
+  margin: 2px auto;
+}
+
+.accordion-item {
+  border: 2px solid #f2f2f2;
+  color: aliceblue;
+  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+  justify-content: space-between;
+  align-items: center;
+  margin: 1rem 0;
+  
+}
+
+.accordion-item-header {
+  padding: 0.5rem 3rem 0.5rem 1rem;
+  min-height: 3.5rem;
+  line-height: 1.25rem;
+  font-weight: bold;
+  display: flex;
+  align-items: center;
+  position: relative;
+  cursor: pointer;
+  }
+
+  .accordion-item-header::after {
+    content: "\002B";
+    font-size: 2rem;
+    position: absolute;
+    right: 1rem;
+    transition: transform 0.2 ease-in-out;
+  }
+
+  .accordion-item-header.active::after {
+    content:"\2212";
+    transform: rotate(-180deg);
+  }
+
+  .accordion-item-body {
+    display: none;
+    padding: 1rem;
+    line-height: 1.5rem;
+    border-top: 1px solid #9aedfc;
+    border-image: linear-gradient(to right, transparent, #9aedfc, transparent) 1;
+  }
+  .accordion-item-header.active + .accordion-item-body {
+    display: block;
+  }
+
```

Comment:
Good job! If you want a smoother transition you can always check the Inspector tool in your browser to see where the transition should be, also some reading here:
http://css3.bradshawenterprises.com/accordions/